src/main/scala/io/iohk/ethereum/blockchain/sync/fast/SyncStateScheduler.scala

```diff
@@ -209,7 +210,8 @@ class SyncStateScheduler(blockchain: Blockchain, bloomFilter: LoadableBloomFilte
       }
 
     case n: BranchNode =>
-      Right(n.children.collect { case HashNode(childHash) =>
+      val children = ArraySeq.unsafeWrapArray(n.children)
```

Comment:
Why is this wrapping needed?

Reply:
`Array` is no longer implicitly converted to `Seq` in 2.13
Without a warning, at least.